pkg/kubelet/dockertools/manager.go

```diff
@@ -1450,7 +1458,20 @@ func (dm *DockerManager) runContainerInPod(pod *api.Pod, container *api.Containe
 	if usesHostNetwork(pod) {
 		utsMode = namespaceModeHost
 	}
-	id, err := dm.runContainer(pod, container, opts, ref, netMode, ipcMode, utsMode, pidMode, restartCount)
+
+	// Set OOM score of the container based on the priority of the container.
```

Comment:
At the least, the oom score calculation could be moved outside to a separate method.